.github/workflows/ci.yml

```
@@ -8,4 +8,4 @@ jobs:
     - uses: ruby/setup-ruby@v1
       with:
         bundler-cache: true # runs 'bundle install' and caches installed gems automatically
-    - run: bundle exec rake
\ No newline at end of file
+    - run: bundle exec rspec
```

```suggestion
    - run: bundle exec rspec

```

newline at eol is missing